README.md

```diff
@@ -18,14 +119,16 @@ pushes to `main`.  To build it locally:
 1. Install the requirements: `cargo install mdbook mdbook-katex mdbook-mermaid`
 2. To continuously build and serve the documentation: `mdbook serve`
 
-## Running Penumbra
+## Running a Penumbra node
```

Comment:
It would be good for this to have a note that this section is for development, not for running nodes that are part of the public testnet, and that we won't be ready for multiple testnet nodes until we reach our MVP2 milestone implementing staking and delegation.

Reply:
Done.